en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/inter-component-communication.md

```diff
@@ -0,0 +1,48 @@
+# Communnication between the Components
+
+Communication between internal components of Choreo Connect (adapter, enforcer, router and apim control plane) are secured via mutual ssl.
+
+Each component has it's private key and truststore and it's configured in against each component tag `config.toml` file as below.
+
+```toml
+    [adapter.keystore] 
+    certPath = "/home/wso2/security/keystore/mg.pem"
+    keyPath = "/home/wso2/security/keystore/mg.key"
+
+    [adapter.truststore]
+    location = "/home/wso2/security/truststore"
+```
+
+## Adding a Certificate to Adapter/ Enforcer Truststore
+
+The trusted certificate location is configured as a volume mount for Adapter and Enforcer components in the docker-compose.yaml file as below.
```

Comment:
This one actually copies both key-cert pair and trust store. 
needs to specifically say that `security/truststore` directory.